src/contents/en-gb/doc_dotnet.md

```diff
@@ -0,0 +1,326 @@
+    },
+    Containers = new List<ContainerPermissions> 
+    { 
+        new ContainerPermissions 
+        { 
+            ContName = "_publicNames", Access = { Insert = true, Update = true, Delete = true} 
+        }
+    }
+};
+```
+***Note:** Passing a list of `ContainerPermissions` allows the definition of various permissions provided to specific containers.*
+
+**5. Send `AuthReq` to the Authenticator**
+
+Once the `AuthReq` instance is initialised it is then used to generate the authorisation request and send it to the Authenticator using the following code:
+
+```csharp
+var encodedReq = await Session.EncodeAuthReqAsync(authReq);
+var url = $"safe-auth://{encodedReq}";
+System.Diagnostics.Process.Start(url);
+```
+**6. Grant access**
+
+Once the authorisation request is received the Authenticator launches and a pop-up dialogue prompts for access (Allow or Deny). The Authenticator then sends the response back to the application.
+
+Once the authorisation request is received, the Authenticator sends an authorisation response using the system URI mechanism. The system matches the URI scheme to the application and launches the application passing the URI as an argument.
+
+.Net system_uri package will be a future release to achieve this on all platforms.
+
+Note: The demo application already has the code to send a response from the new instance to the already running console app instance using [IPC Pipes](https://docs.microsoft.com/en-us/dotnet/standard/io/pipe-operations).
+
+Once a response is received, it is decoded and checked whether the request was granted or denied. If the request was granted a new session is initialised to communicate with SAFE Network:
+```csharp
+var decodeResult = await Session.DecodeIpcMessageAsync(encodedRequest);
+if (decodeResult.GetType() == typeof(AuthIpcMsg))
+{
+    var ipcMsg = decodeResult as AuthIpcMsg;
+    //Create session object
+    if (ipcMsg != null)
+    {
+        var session = await Session.AppRegisteredAsync(AppId, ipcMsg.AuthGranted);
+    }
+}
+else
+{
+    Console.WriteLine("Auth Request is not Granted");
+}
+```
+
+
+## Connecting to the live network
+
+The process to connect to live is the same as mock except:
+- The non-mock SAFE browser is used as indicated by its title `safe-browser-<version>-<platform>.zip` (see pre-requisites for download instructions).
+- The authentication process is against the live network and not the MockVault file.
+
+**1. Download the SAFE (non-mock) browser** 
+
+If you haven't already done so, download the SAFE (non-mock) Browser (see pre-requisites for instructions)
+
+**2. Log in**
+
+Log in using your existing SAFE Network account or create an account from the Authenticator.
+
+Keep the browser open and logged in to your account before proceeding with next steps.
+
+**3. Authentication code**
+
+Authentication steps are the same as mentioned in mock authentication using SAFE mock browser. So, Follow steps 4-6 given in previous section.

```

Comment:
`Follow steps 4-6 in the previous section as`~A~ `a`uthentication steps are the same as mentioned in mock authentication using `the` SAFE mock browser. ~So, Follow steps 4-6 given in previous section.~